src/biip/gs1/_messages.py

```diff
@@ -4,10 +4,12 @@
 
 from dataclasses import dataclass
 from typing import Iterable, List, Optional, Union
+import re
```

Comment:
This should make isort happy:

```suggestion
import re
from dataclasses import dataclass
from typing import Iterable, List, Optional, Union
```